### ISSUE TITLE
The Jenkins bot can trigger itself.

### ISSUE THREAD

**Issue by nicolasnoble** [janinko/ghprb#437]
Somebody on our team changed the default wording for the Jenkins bot from "Can one of the admin verify this patch" to something more verbose, including the words that went basically like "one of our developers will have to say 'ok to test' for this PR to be tested on Jenkins".

Unfortunately, the Jenkins bot itself, in order to have the required permissions for updating the status of the pull request, is on the group of users that is authorized to start a test.

So basically, we ended up in a situation where the bot would comment that one of the admins would have to say "ok to test", and that comment itself would be sufficient to trigger the test.

Suggestion for fixing: the Jenkins username itself should be excluded from the list of users authorized to trigger a test.

**Comment by elasticdog** [comment]
I have also been bitten by this behavior in the past, which was a security concern as any non-authorized user that opened a PR had their commits tested automatically.

**Comment by nicolasnoble** [comment]
Alternative suggestion: show a warning in the configuration if the custom sentence matches any of the regular expressions.
